crates/bevy_render/src/camera/visible_entities.rs

```diff
@@ -204,8 +204,8 @@ pub fn visible_entities_system(
         &mut VisibleEntities,
         Option<&RenderLayers>,
     )>,
-    visible_query: Query<(Entity, &Visible, Option<&RenderLayers>)>,
-    visible_transform_query: Query<&GlobalTransform, With<Visible>>,
+    visible_query: Query<(Entity, &Visible, Option<&RenderLayers>), With<WithinFrustum>>,
```

Comment:
As-is this completely breaks all 3d rendering (and any custom non-sprite rendering) because `WithinFrustum` is never added to those entities. We should probably add another system that adds `WithinFrustum` to every  `Query<Entity, (With<MainPass>, Without<Sprite>, Without<TextureAtlasSprite>)>`

Reply:
I added the suggested system to frustum_culling.rs.
the 3d_scene example does run as it should.

Reply:
Hmm this is a bit problematic because it _requires_ the SpritePlugin to draw anything. Thats a bit too hack-ey for my taste. Sprites should be a layer on top of rendering, not a required piece. Thinking on this now.

Reply:
I think maybe switching to a `Without<OutsideFrustum>` filter instead might do the trick.